middle_end/flambda/inlining/inlining_cost.ml

```diff
@@ -502,16 +508,24 @@ module Threshold = struct
       Can_inline_if_no_larger_than (min i1 i2)
 end
 
-let smaller denv lam ~than =
-  smaller' denv lam ~than <> None
+module Inline_res = struct
+
+  type t =
+    | Cannot_inline
+    | Can_inline of int
+    (* size of the inlinable expression, used in inlining reports *)
+
+end
```

Comment:
If you only define a type, I'd rather not have the module around it. If you need to add more functions you can always bring the module back.

Reply:
I've removed the modules

Reply:
If you wish to keep this module, fine, but replying "I've removed the modules" at the only place you haven't feels a bit misleading...

Reply:
Ah.. sorry I forgot about that one

Reply:
Fixed, ^^